Source/WebCore/html/ImageDocument.cpp

```diff
@@ -147,6 +147,9 @@ void ImageDocument::updateDuringParsing()
     if (!m_imageElement)
         createDocumentStructure();
 
+    if (RefPtr frame = document().frame(); !frame)
```

Comment:
You don’t need to ref the frame just to null check it.

`if (!document().frame())`